### ISSUE TITLE
@import linter

### ISSUE THREAD

**Issue by davidtheclark** [brigade/scss-lint#266]
Feature request:

Simple linter that would check `@import` statements for two things:
- the partial's leading underscore is omitted
- the partial's file extension is omitted

Bad:

``` scss
@import "foo/_bar.scss";
@import "_bar.scss";
@import "_bar";
@import "bar.scss";
```

Good:

``` scss
@import "foo/bar";
@import "bar";
```

Or with options you could _force_ the leading underscore and trailing extension to be included.

Any objections?
